sphinx_book_theme/__init__.py

```diff
@@ -22,6 +22,13 @@ def add_static_path(app):
     static_path = Path(__file__).parent.joinpath("static").absolute()
     app.config.html_static_path.append(str(static_path))
 
+    # Compile the css file if it's not been compiled already
+    compiled_css_file = static_path / "sphinx-book-theme.css"
+    if not compiled_css_file.exists():
```

Comment:
```suggestion
    compiled_css_file = static_path / "sphinx-book-theme.css"
    compiled_css_file2 = static_path / "_print.css"
    if not (compiled_css_file.exists() and compiled_css_file2.exists()):
```

just to be safe?

Reply:
That file won't get compiled into a css. That's what the `_` does. FWIW, I am borderline wondering if I should break out the main scss file into small `_sidebar` and friends. :)

Reply:
Relevant Stack-Overflow question to save you from a round of web searches: https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css

Reply:
Ah cheers; just showing by "newbness" on SCSS

Reply:
So does this mean that if we change the scss file we should delete the css file so it is recompiled?

Reply:
You should if you're using `nox -s docs-live` or `nox -s docs`, which do recompile the SCSS.

The only thing this exists for, is to be able to install+use this package off VCS. :)